CHANGES-guidelines.md

```diff
@@ -0,0 +1,52 @@
+Major releases (e.g. 1.13.0) are treated different from patch releases (e.g. 1.13.3). We would like to focus on building up complete and correct changelog content especially for patch releases. 
+
+## Patch releases
+
+**Any user-facing change** between two patch releases must be reflected in `CHANGES.md`, in one of three sections:
+* "Notable changes"
+* "Fixed and improved"
+* "Security updates"
+
+### How to choose the section 
```

Comment:
As someone who regularly bumps external packages that have security considerations, I still find this confusing.

I'd suggest that we re-order the CHANGELOG with Security Updates first, and use these criteria:
- If a change fixes a security issue and for this reason it is strongly recommended to stop using earlier versions of this release, then the change must go in _Security updates_.
- If your patch adds a change to the installation procedure or a new DC/OS install-time configuration..._Notable changes_
- If your patch adds a completely new feature..._Notable changes_
- If you update a component with externally maintained release notes..._Notable changes_
- Other external facing changes go in _Fixed and Improved_

Reply:
Feel free to update it @jongiddy , my main goal is to get first rid of this wiki page https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines, so that we can disable the wiki function on github.

Reply:
I made the change as the separate pull request https://github.com/dcos/dcos/pull/7232 since only admins can modify https://github.com/dcos/dcos branches.